Sources/SwiftSyntax/Trivia.swift

```diff
@@ -42,6 +42,68 @@ public struct Trivia: Sendable {
     pieces.isEmpty
   }
 
+  /// The string contents of all the comment pieces with any comments tokens trimmed.
+  ///
+  /// Each element in the array is the trimmed contents of a line comment, or, in the case of a multi-line comment a trimmed, concatenated single string.
+  public var commentValues: [String] {
+    var comments = [String]()
+    var partialComments = [String]()
+
+    var foundStartOfCodeBlock = false
+    var foundEndOfCodeBlock = false
+    var isInCodeBlock: Bool { foundStartOfCodeBlock && !foundEndOfCodeBlock }
+
+    for piece in pieces {
+      switch piece {
+      case .blockComment(let text), .docBlockComment(let text):
+        let text = text.trimmingCharacters(in: "\n")
+
+        foundStartOfCodeBlock = text.hasPrefix("/*")
+        foundEndOfCodeBlock = text.hasSuffix("*/")
+
+        let sanitized =
+          text
+          .split(separator: "\n")
+          .map { $0.trimmingAnyCharacters(in: "/*").trimmingAnyCharacters(in: " ") }
```

Comment:
I think this would also trim any slashes at the start of a line. And I don’t think we want that. Eg. in

```swift
/* 
/abc
*/
```

I would expect the comment value to be `/`.

I would just trim the following:
- At the start of the comment: `/*` or `/**`
- Inside the block comment: Any number of spaces followed by an optional `*`
- At the end of the comment: `*/`